.gitignore

```diff
@@ -7,3 +7,4 @@
 *.dylib
 *.pro.user
 Makefile
+CMakeLists.txt.user
```

Comment:
Heh, use full filname in commit message :).

Reply:
Fixed.